src/Layout.js

```diff
@@ -136,11 +136,11 @@ var computeLayout = (function() {
       case 'column-reverse': value = node.style.marginTop;    break;
     }
 
-    if (value != null) {
+    if (value !== null && typeof value !== 'undefined') {
```

Comment:
You should do

value !== undefined

The typeof check is when the variable may or may not exist. This is useful for checking globals. But in this case we know it exists so we can just check against the undefined value.